CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## [UNRELEASED]
+### Added
+- Added a changelog
+
+### Changed
+- create orders without a contract_id
+- add get operation for orders
```

Comment:
I think http://keepachangelog.com/ makes a few good points regarding "What makes a good change log?" - and one of them is

> * Each version should:
>   * List its release date in the above format.
>   * Group changes to describe their impact on the project, as follows:
>   * **```Added``` for new features.**
>   * ```Changed``` for changes in existing functionality.
>   * ```Deprecated``` for once-stable features removed in upcoming releases.
>   * ```Removed``` for deprecated features removed in this release.
>   * ```Fixed``` for any bug fixes.
>   * ```Security``` to invite users to upgrade in case of vulnerabilities.

So I would prefer the second meaning. As a developer reviewing a changelog: Are you interested in which new files where added or which features?

@andrecedik, @jannik-hh, @sssggr please weigh in here.